src/Controller/Admin/OrderOverview.php

```diff
@@ -21,6 +21,8 @@
 
 class OrderOverview extends OrderOverview_parent
 {
+    protected $captureStatus = null;
```

Comment:
i think a comment like /* @var null|string $captureStatus */ would improve the readability